### DISCUSSION TITLE
Move to webdoc for documentation generation

### DISCUSSION THREAD

**Discussion by parasyte** [melonjs/melonJS#1028] [Ideas]
I got sick of using JSDoc3 in one of my personal projects, so I made my own: http://blog.kodewerx.org/2014/05/trials-of-documenting-javascript-source.html

Even if we don't use that, we should definitely replace JSDoc3 with something better. This has been a problem forever, and mentioned a few times in #66.

**Comment by obiot** [comment]
at least we have a link to the 1.1.0 release ticket :)

**Comment by obiot** [comment]
after 3 Years, there is no intention anymore to move away from JSDoc 3, I'm statisfied with it, and I'd rather just change the template (https://nhnent.github.io/tui.jsdoc-template/latest/) to improve readability.

**Comment by obiot** [comment]
reopening this one 

https://github.com/webdoc-js/webdoc

https://medium.com/javascript-in-plain-english/webdoc-the-next-generation-documentation-tool-3e32b5f50c3e

**Comment by obiot** [comment]
this one is being revived with #1089
